models/event.js

```diff
@@ -0,0 +1,46 @@
+let Event = null;
+const mongoose = require('mongoose');
+const connString = 'mongodb://localhost:27017/events';
+mongoose.connect(connString);
+
+mongoose.connection.on('connected',  () => {
+  console.log('Mongoose default connection open to ' + connString);
+});
+
+// If the connection throws an error
+mongoose.connection.on('error', (err) => {
+  console.log('Mongoose default connection error: ' + err);
+});
+
+// When the connection is disconnected
+mongoose.connection.on('disconnected', () => {
+  console.log('Mongoose default connection disconnected');
+});
+
+const Schema = mongoose.Schema;
+const eventSchema = new Schema(
+  {
+    _id: { type: Number, required: true },
+    title: { type: String, required: true, index: true},
+    description: { type: String, required: true },
+    date: { type: Date }
+  },
+  {
+    versionKey: false
```

Comment:
Hi Ariel, we added it to remove the field "__v" from documents, because we thought we might not need that field, and also I saw the use of that sentence in the [article](https://scotch.io/tutorials/test-a-node-restful-api-with-mocha-and-chai) suggested in Task #5 of Milestone 4: Testing your API.

Now I'm reading in [this guide](http://mongoosejs.com/docs/guide.html) that it's not advisable to disable versioning.